bumble/gatt.py

```diff
@@ -219,9 +227,10 @@ def __init__(self, uuid, characteristics: list[Characteristic], primary=True):
             uuid.to_pdu_bytes(),
         )
         self.uuid = uuid
-        # self.included_services = []
+        self.included_services = included_services
         self.characteristics = characteristics[:]
         self.primary = primary
+        self.registered = False
```

Comment:
I think it would be better to keep track of that state in the server rather than in the service itself. It's a bit dangerous to have `Service` objects be mutable by the server. They really should behave like read-only objects. There's maybe a small performance penalty in keep the state out of the object, but since a GATT DB really can't get all that large, that would never be an issue in practice.

Reply:
Done